docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.9"
+    
+services:
+  waltz:
+    build: ./
+    restart: always
+    container_name: waltz
+    hostname: waltz
+    command: ["update","run"]
+    ports:
+      - "8080:8080"
+    depends_on:
+      - postgres
+    links:
+      - postgres
+    environment:
+      DB_HOST: postgres
+      DB_PORT: 5432
+      DB_NAME: waltz
+      DB_USER: waltz
+      DB_PASSWORD: waltz
+      DB_SCHEME: waltz
+      WALTZ_FROM_EMAIL: "help@finos.org"
+      WALTZ_BASE_URL: "http://127.0.0.1:8080/"
+      CHANGELOG_FILE: "/opt/waltz/liquibase/db.changelog-master.xml"
```

Comment:
anything in there that is the default should probably be removed to simplify the `docker-compose.yml` file